sql/moz-fx-data-shared-prod/telemetry/releases_latest/view.sql

```diff
@@ -0,0 +1,89 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.latest_releases`
```

Comment:
```suggestion
  `moz-fx-data-shared-prod.telemetry.releases_latest`
```
Make sure the directory name matches the view name

Reply:
oops 😬 good catch. Thanks!